test/runner.sh

```diff
@@ -59,9 +59,9 @@ trap cleanup EXIT
 # we use mkdir here because it is an atomic operation unlike existance of a lockfile
 # where creating and checking are 2 separate operations
 if mkdir ${TEST_OUTPUT_DIR}/.pg_init 2>/dev/null; then
-  cat <<EOF | ${PSQL} "$@" -U ${USER} -d template1 -v ECHO=none >/dev/null 2>&1
+  ${PSQL} "$@" -U ${USER} -d template1 -v ECHO=none >/dev/null 2>&1 <<EOF
     SET client_min_messages=ERROR;
-    GRANT CREATE ON SCHEMA public TO PUBLIC;
+    GRANT ALL ON SCHEMA public TO PUBLIC;
```

Comment:
```suggestion
    GRANT CREATE ON SCHEMA public TO ${TEST_PGUSER};
    GRANT CREATE ON SCHEMA public TO ${TEST_ROLE_DEFAULT_PERM_USER};
    GRANT CREATE ON SCHEMA public TO ${TEST_ROLE_DEFAULT_PERM_USER_2};
    GRANT CREATE ON SCHEMA public TO ${TEST_ROLE_1};
    GRANT CREATE ON SCHEMA public TO ${TEST_ROLE_2};
    GRANT CREATE ON SCHEMA public TO ${TEST_ROLE_3};
```
You are still modifying the default ACL here which I think should be avoided and instead we should adjust the permissions of the actual users we use here.